### ISSUE TITLE
Supporting MIP solution pool

### ISSUE THREAD

**Issue by th-yoon** [ERGO-Code/HiGHS#1193]
In SCIP or Gurobi, I can set the maximum number of solutions to store in the model, and I can retrieve the solutions that are not necessarily optimal.
It'd be great if similar functionality is supported in HiGHS.
Thanks for developing this high quality open source MIP/LP solver.

**Comment by chriscoey** [comment]
Hi @jajhall, just wondering if this functionality is exposed yet? Is there an example or anything showing how to use it?

**Comment by jajhall** [comment]
> Hi [@jajhall](https://github.com/jajhall), just wondering if this functionality is exposed yet? Is there an example or anything showing how to use it?

Yes, with with the "MIP (improving) solution" callbacks, or by setting the `mip_improving_solution_save` option to be true and extracting them, afterwards with `Highs::getSavedMipSolutions()`.

What language are you calling from?

**Comment by chriscoey** [comment]
Great! We are using HiGHS from Julia, through MOI/HiGHS.jl. Thanks.

**Comment by jajhall** [comment]
Then I guess that there are Julia examples in JuMP. We don't have Julia examples because @odow does such a good job!
